esx_service/vmdk_ops.py

```diff
@@ -606,6 +606,13 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
 
     elif cmd == "remove":
         response = removeVMDK(vmdk_path)
+        # remove succeed, remove infomation of this volume from volumes table
```

Comment:
If a volume is created by a VM which is not associated with any tenant(the logic of __FALLBACK tenant does not exist yet), we should not insert it to the "volumes" table. Right?